mitodl/open-discussions
# fix mobile styling of dialog

#### Pre-Flight checklist


#### What are the relevant tickets?

just noticed we have an issue with the dialog on mobile, see screenshots below. CSS only fix.

#### What's this PR do?

fixes an issue where the top of the dialog was getting cut off by our top bar on mobile (or on any short device really).

#### How should this be manually tested?

confirm the issue on master, then check this out and confirm it fixes it without doing anything too weird.


#### Screenshots (if appropriate)

before:

![fixedthedialogfff](https://user-images.githubusercontent.com/6207644/69995767-b6e6c280-151e-11ea-9d22-8b3a722c6d4c.png)


after:

![fixedthedialog](https://user-images.githubusercontent.com/6207644/69995730-a46c8900-151e-11ea-9c6f-84fdfe857906.png)
